tests/yadism/cf/test_cfh_pol.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+import pytest
+from eko.thresholds import ThresholdsAtlas
+
+from yadism import observable_name as obs
+from yadism.coefficient_functions.heavy import f2_nc as h_f2_nc
+from yadism.coefficient_functions.heavy import f3_nc as h_f3_nc
+from yadism.coefficient_functions.heavy import fl_nc as h_fl_nc
+from yadism.coefficient_functions.heavy import g1_nc as h_g1_nc
+from yadism.coefficient_functions.heavy import g4_nc as h_g4_nc
+from yadism.coefficient_functions.heavy import gl_nc as h_gl_nc
+
+
+class MockCouplingConstants:
+    def get_weight(self, _pid, _q2, qct):
+        if qct == "VV":
+            return 1
+        if qct == "VA":  # -->  must be 0 due to symmetry reasons
+            return 2
+        if qct == "AV":  # -->  must be 0 due to symmetry reasons
+            return 4
+        if qct == "AA":
+            return 8
+        raise ValueError(f"Unkown {qct}")
+
+
+class MockSF:
+    def __init__(self, n):
+        self.obs_name = obs.ObservableName(n)
+        self.coupling_constants = MockCouplingConstants()
+        self.m2hq = [1.0, 2.0, 3.0]
+        self.threshold = ThresholdsAtlas(self.m2hq)
+
+
+class MockESF:
+    def __init__(self, sf, x, Q2):
+        self.sf = MockSF(sf)
+        self.x = x
+        self.Q2 = Q2
+        self.process = "NC"
+
+
+def test_cg_NLO():
+    Q2 = 200
+    esf = MockESF("g1_charm", 0.1, Q2)
+    m2hq = 2
+    for nf in [3, 4]:
+        for z in [1e-1, 1e-2, 1e-3]:
+            cg_1 = h_g1_nc.GluonVV(esf, nf, m2hq=m2hq)
+            cg_2 = h_g1_nc.GluonAA(esf, nf, m2hq=m2hq)
+            order = lambda pc, o="NLO": pc.__getattribute__("NLO")()
+            a_1 = order(cg_1).reg(z, order(cg_1).args["reg"])
+            a_2 = order(cg_2).reg(z, order(cg_2).args["reg"])
+            np.testing.assert_allclose(
+                a_1,
+                a_2,
+                err_msg="Axial vector-axial vector Coefficients and vector-vector coefficients at NLO should be the same, but are not",
+            )
+
+
+@pytest.mark.skip
```

Comment:
why this test is skipped? Is it because it's taking too much?

Reply:
This relation is not valid for massive so it was skipped, but now it has been removed in [017950b](https://github.com/NNPDF/yadism/pull/185/commits/017950b21bcec7c0a4dace2e79165cbf4f041a62).